src/Imath/ImathFrame.h

```diff
@@ -3,33 +3,38 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+///
+/// @file  ImathFrame.h
+///
+/// @brief Functions for computing reference frames.
+///
+/// These methods compute a set of reference frames, defined by their
+/// transformation matrix, along a curve. It is designed so that the
+/// array of points and the array of matrices used to fetch these
+/// routines don't need to be ordered as the curve.
+///
```

Comment:
These comments will be lost. When you do a pure Doxygen run (to html), you end up with a "file by file" index (as well as class by class), and this stuff from `@file` will go to introduce each file. But the way you've arranged the docs through breathe and sphinx (which I approve), you are making class by class documentation, so the Doxygen comments on the files themselves will never show up anywhere.

I recommend transferring these `@file` comments to appropriate parts of the docs for the class themselves, or for just a collection of functions (not in a class), use a "group" appropriately to cluster functions and then these can become the group comments.

Reply:
I removed the @file/@brief directives and left the introductory comments in non-doxygen style for the reader of the headers, and moved info to the appropriate to the class/functions as wel.